composer.json

```
@@ -22,6 +22,7 @@
         "doctrine/doctrine-bundle": "^2.10",
         "doctrine/orm": "^2.15",
         "knplabs/knp-time-bundle": "^1.20|^2.0",
+        "liip/monitor-bundle": "3.x-dev",
```

```suggestion
        "liip/monitor-bundle": "^3.0",
```

This should work because of the minimum-stability flag below.